Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+namespace Algorithms.Sequences
+{
+    /// <summary>
+    ///     <para>
+    ///         Sequence of number of primes less than 10^n (with at most n digits).
+    ///     </para>
+    ///     <para>
+    ///         Wikipedia: https://wikipedia.org/wiki/Prime-counting_function.
+    ///     </para>
+    ///     <para>
+    ///         OEIS: https://oeis.org/A006880.
+    ///     </para>
+    /// </summary>
+    public class NumberOfPrimesByPowersOf10Sequence : ISequence
+    {
+        /// <summary>
+        /// Gets sequence of numbers of primes.
+        /// </summary>
+        public IEnumerable<BigInteger> Sequence
+        {
+            get
+            {
+                ISequence primes = new PrimesSequence();
+                var prevPowerOf10 = new BigInteger(0);
```

Comment:
0 is not a power of 10

Reply:
Variable has been removed during refactoring.